tests/_test_utils/test_coordinate.py

```diff
@@ -24,11 +24,12 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 """
+
 import logging
 import datetime
 
 import numpy as np
-import pytest
+import pytest # type: ignore
 
```

Comment:
this type: ignore is new here. What is it doing, just my interests in it. What is the reason for this?

the linter will likly  want to blanks to seperate the  comment

Reply:
I think that is not good to add it here. When we have type related issues, we won't see that now, correnct?